src/assignment/types.ts

```diff
@@ -0,0 +1,32 @@
+import { CoreContentType } from '@viaa/avo2-types/types/core/content';
+
```

Comment:
Past dit uiteindelijk eigenlijk niet best in onze typings repo ook zodat we die hier in de toekomst kunnen weghalen? Of is er een bepaalde reden dat deze hier zitten en niet daar?